generator/generator_tool/generator_tool.cpp

```diff
@@ -80,6 +85,14 @@ DEFINE_bool(disable_cross_mwm_progress, false,
 DEFINE_string(srtm_path, "",
               "Path to srtm directory. If set, generates a section with altitude information "
               "about roads.");
+DEFINE_string(countries_dir, "",
+              "Directory with countries file which describes mwm tree. Used for routing and cross mwm "
+              "routing generation to get country specific routing restrictions (we need to know which "
+              "country or region does this mwm belong).");
+DEFINE_string(countries_filename, "",
```

Comment:
If countries_filename is empty I use storage::Storage default constructor which knows the default name of countries file. I think it's good to have a single place to store default value.